hasher-matcher-actioner/hmalib/lambdas/actions/action_evaluator.py

```diff
@@ -54,7 +53,6 @@ def get(cls):
         HMAConfig.initialize(os.environ["CONFIG_TABLE_NAME"])
```

Comment:
env values used in the API lambda but not declared in the top level module are hard to debug and easy to miss. We've gotten around this before by having this init occur within api_root.py. Thoughts?

Reply:
Bit confused. Don't think this is called directly from an api call. This is called after a match occurs?

Reply:
Ah my mistake I missed this files path on first review. 
Edit: env values used in the ~~API~~ *any* lambda but not declared in the top level module are hard to debug and easy to miss....

Ideally imo this would be a constant declaration at the top of the file like in `api_root.py` (just for ease of refactor later and quick checks between the file and tf) but because this file as a `lambda_handler` method and something like this already being done in `fetcher.py` it is probably fine.